CMakeLists.txt

```diff
@@ -22,6 +22,9 @@ if(NOT CMAKE_BUILD_TYPE AND NOT CMAKE_CONFIGURATION_TYPES)
       STRING "Choose the type of build." FORCE)
 endif()
 
+# set std 11
+set (CMAKE_CXX_STANDARD 11)
```

Comment:
@lockshaw Have we reached out the agreement that FlexFlow will use c++17 moving forward?